app/templates/admin/sessions/list.hbs

```diff
@@ -0,0 +1,56 @@
+<div class="sixteen wide column">
+  <table class="ui stackable very basic table">
+    <thead>
+      <tr>
+        <th>{{t 'Event'}}</th>
+        <th>{{t 'Title'}}</th>
+        <th>{{t 'Speaker'}}</th>
+        <th>{{t 'Date of Submission'}}</th>
+        <th>{{t 'Start Date'}}</th>
+        <th>{{t 'End Date'}}</th>
+        <th>{{t 'options'}}</th>
+      </tr>
```

Comment:
options should start with the capital letter i.e Options as it is one of the headings.